### PR TITLE
ACRS-59 ACRS-69 Implement /child-details /children-summary

### PR DESCRIPTION
**Pull request by michaelhartddat**
## What? 

[ACRS-59 Create 'Child's details' page](https://collaboration.homeoffice.gov.uk/jira/browse/ABCD-59)
[ACRS-69 Create a 'Children you are referring' summary page](https://collaboration.homeoffice.gov.uk/jira/browse/ACRS-69)

Adds the /child-details & /children-summary pages with looping aggregator limit of 100.

### CODE REVIEW

File: apps/acrs/index.js [hunk 1]
```diff
@@ -13,13 +13,18 @@ const FamilyMemberBahaviour = require('./behaviours/family-member');
...
 const LimitParents = require('./behaviours/limit-parents');
 const ResetSummary = require('./behaviours/reset-summary');
 const ModifySummaryChangeLinks = require('./behaviours/summary-modify-change-link');
+const ChildrenSummary = require('./behaviours/children-summary');
+const LimitChildren = require('./behaviours/limit-children');
+
 
 // Aggregator section limits
 const PARENT_LIMIT = 2;
+const CHILDREN_LIMIT = process.env.NODE_ENV === 'development' ? 2 : 100;
```
Comment (jamiecarterHO) [inline apps/acrs/index.js:27]: Nice

File: apps/acrs/index.js [hunk 2]
```diff
@@ -13,13 +13,18 @@ const FamilyMemberBahaviour = require('./behaviours/family-member');
 const FamilyDetailBahaviour = require('./behaviours/get-family-detail');
 const Locals18Flag = require('./behaviours/locals-18-flag');
 const AggregateSaveUpdate = require('./behaviours/aggregator-save-update');
+const AggregatorSaveUpdate = AggregateSaveUpdate;
```
Comment (jamiecarterHO) [inline apps/acrs/index.js:16]: Didn't like the name? :)

File: apps/acrs/sections/summary-data-sections.js [hunk 3]
```diff
@@ -185,5 +185,29 @@ module.exports = {
...
+      },
+      {
+        steps: '/child-details',
+        field: 'child-living-situation'
+      },
+      {
+        steps: '/child-details',
+        field: 'child-why-without-child'
+      }
+    ]
```
Comment (jamiecarterHO) [inline apps/acrs/sections/summary-data-sections.js:211]: If you want to show the aggregated children on the summary pages as I have with parents you can check lines 131-156 above.

You might want to deal with it later though

File: apps/acrs/index.js [hunk 4]
```diff
@@ -297,28 +302,78 @@ module.exports = {
...
+        'child-why-without-child'
+      ],
+      aggregateLimit: CHILDREN_LIMIT,
+      titleField: 'child-full-name',
+      addStep: 'child-details',
+      addAnotherLinkText: 'child',
+      locals: {
+        showSaveAndExit: true,
+        referredChildrenLimit: CHILDREN_LIMIT
+      },
```
Comment (jamiecarterHO) [inline apps/acrs/index.js:373]: good

**Review by jamiecarterHO — APPROVED** [review]
Good stuff. Couple of questions, but it will work just fine either way

File: apps/acrs/sections/summary-data-sections.js (hunk 3 above)
Comment (michaelhartddat) [reply apps/acrs/sections/summary-data-sections.js]: Implemented as indicated, good call.

**Pull request merged**